### PR TITLE
Resolve comments, update EF7 to latest beta6 build.

### PR DESCRIPTION
**Pull request by rayao**
Some latest configuration API changes in EF7 will break RESTier.EF7 build, this change contains fixes.

### CODE REVIEW

File: src/Microsoft.Restier.Samples.Northwind/Models/NorthwindContext.cs [hunk 1]
```diff
@@ -17,18 +19,18 @@ public NorthwindContext()
         {
             try
             {
-                if (!Database.AsRelational().Exists())
+                if (!this.Database.GetService<IRelationalDatabaseCreator>().Exists())
```
Comment (congysu) [inline src/Microsoft.Restier.Samples.Northwind/Models/NorthwindContext.cs]: nit: `this` is optional.

File: src/Microsoft.Restier.Samples.Northwind/Models/NorthwindContext.cs [hunk 2]
```diff
@@ -17,18 +19,18 @@ public NorthwindContext()
...
             }
             catch
             {
-                ResetDataSource();
+                LoadDataSource();
             }
         }
 
-        protected override void OnConfiguring(EntityOptionsBuilder optionsBuilder)
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
```
Comment (congysu) [inline src/Microsoft.Restier.Samples.Northwind/Models/NorthwindContext.cs]: Is this a breaking change?

**Comment by congysu** [comment]
Looks good to me.

File: src/Microsoft.Restier.Samples.Northwind/Models/NorthwindContext.cs (hunk 2 above)
Comment (rayao) [inline src/Microsoft.Restier.Samples.Northwind/Models/NorthwindContext.cs]: Yeah, it seems breaking changes keep coming in.

**Comment by rayao** [comment]
Sorry guys, for messing this up again.
Please ignore this pull request since they're still actively introducing breaking changes into EF7.
It's unbelievable to me, isn't VS2015 shipping in weeks?

File: src/Microsoft.Restier.Samples.Northwind/Models/NorthwindContext.cs (hunk 1 above)
Comment (rayao) [inline src/Microsoft.Restier.Samples.Northwind/Models/NorthwindContext.cs]: Thanks.

**Comment by karataliu** [comment]
I've created https://github.com/OData/RESTier/pull/247 for upgrading to beta8 based on your pull request.

Looks like we can close this one now, thanks a lot for the contribution!

**Pull request closed without merging**